.github/labeler.yml

```diff
@@ -126,6 +126,11 @@ setup payload:
 ############################################################
 platform:
     - src/platform/*
+    - config/tizen/chip-gn/platform
+    - examples/platform
+    - scripts/tools/memory/platform
+    - src/include/platform
+    - src/lib/dnssd/platform/*
```

Comment:
why does this have a * and not the rest?

Reply:
Error :)